olp-cpp-sdk-core/src/cache/DefaultCache.cpp

```diff
@@ -281,7 +288,19 @@ DefaultCache::StorageOpenResult DefaultCache::SetupStorage() {
   if (settings_.max_memory_cache_size > 0) {
     memory_cache_.reset(new InMemoryCache(settings_.max_memory_cache_size));
   }
-  if (settings_.disk_path) {
+
+  // For backwards compatibility.
+  // If the user provided old disk_path instead of a new one, use it.
```

Comment:
I'm not sure about the team's agreement on this topic. So I'm not bikeshedding, rather just asking and thinking out loud.

Is this kind of comment necessary and useful? The comment explains the How part, meanwhile it's preferable for the code comments to explain the Why part of the code and for the code itself to explain the How, isn't it? 

That is. It's obvious from line 299 that we're going to use the deprecated setting, so the comment at line 293 loses its usefulness.

Reply:
Changed the comment.